decomposed/extensions/MDDE_LDM/Profile/Model/Methods/mdde_InitializeExportExtension.xml

```diff
@@ -0,0 +1,99 @@
+<o:MethodTargetItem Id="A2DF8843-2E87-46FC-9263-927B3E02D9E8">
+<a:ObjectID>A2DF8843-2E87-46FC-9263-927B3E02D9E8</a:ObjectID>
+<a:Name>mdde_InitializeExportExtension</a:Name>
+<a:CreationDate>1701423896</a:CreationDate>
+<a:Creator>WillemOtten</a:Creator>
+<a:TargetCategory.Type>1</a:TargetCategory.Type>
+<a:MethodScript>Sub %Method%(obj)
+   &#39;Implement your method on &lt;obj&gt; here
+   &#39;Set success indicator to true
```

Comment:
Maybe elaborate a bit more on why this function exists.

I think the main reason is we can't call templates in another model from GTL, but we can call extended attributes. So in order to get the contents of the template in the export extension there is also an extended attribute in which we can store the output of the template. This function will update all the content extended attributes with the output of the corresponding template on the same metaclass/stereotype.

Reply:
Added explanation in the comment section of the method